src/Appwrite/Auth/Phone/Vonage.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace Appwrite\Auth\Phone;
+
+use Appwrite\Auth\Phone;
+
+// Reference Material
+// https://dashboard.nexmo.com/getting-started/sms
+
+class Vonage extends Phone
+{
+    /**
+     * @var string
+     */
+    private string $endpoint = 'https://rest.nexmo.com/sms/json';
+
+    /**
+     * @param string $from
+     * @param string $to
+     * @param string $message
+     * @return void
+     */
+    public function send(string $from, string $to, string $message): void
+    {
+        $to = ltrim($to, '+');
+        $from = "Appwrite";
```

Comment:
We have a env variable for `_APP_PHONE_FROM` 👍🏻

Reply:
Yes, however, Vonage doesn't need anyone to purchase a phone number unless there are any inbound SMSes. 

In this case, shall I keep this as an optional field that can set to `Appwrite` if the user keeps the environment variable empty?

Reply:
On further research, we do need a phone number to send SMSes in the USA. Let me make the change.